WildShape-Sheets-API/Services/PlayerCharacterService.cs

```diff
@@ -22,6 +22,11 @@ public PlayerCharacterService(IConfiguration configuration, DataBaseService data
 
         public List<PlayerCharacter> GetPlayerCharacters() => _dataBaseService.playerCharacterCollection.Find(pc => true).ToList();
 
+        public List<PlayerCharacter> GetUsersPlayerCharacters(string email) {
+            var user = _userService.GetUserByEmail(email);
+            return user.Characters;
+        }
+
         public PlayerCharacter GetPlayerCharacterById(string id) => _dataBaseService.playerCharacterCollection.Find(pc => pc.Id == id).FirstOrDefault();
 
         public PlayerCharacter? CreatePlayerCharacter(string userEmail, PlayerCharacter pc)
```

Comment:
Pro tip:
`if (user.Characters?.Count <= 4) {`
You don't ever want a hard coded number. Always use a defned constant. MAX_NUMBER_PLAYER_CHARACTERS=4 (or something like that. That way it is clear what 4 is, and it is simple to change all occurances in the code rather that searching for "4" and getting who knows else what.

Reply:
i did know that, it was in my brain but i forgot to do it